docs/introduction.html

```diff
@@ -43,7 +43,7 @@
       <ul style="float: left; width: 40%;">
       <li>The <a href="/documentation.html#producerapi">Producer API</a> allows an application to publish a stream of records to one or more Kafka topics.
       <li>The <a href="/documentation.html#consumerapi">Consumer API</a> allows an application to subscribe to one or more topics and process the stream of records produced to them.
-    <li>The <a href="/documentation.html#streams">Streams API</a> allows an application to act as a <i>stream processor</i>, consuming an input stream from one or more topics and producing an output stream to one or more output topics, effectively transforming the input streams to output streams.
+    <li>The <a href="/documentation/streams">Streams API</a> allows an application to act as a <i>stream processor</i>, consuming an input stream from one or more topics and producing an output stream to one or more output topics, effectively transforming the input streams to output streams.
```

Comment:
Hmm. That is a good question: currently we have top-level `intro.html` which is just a virtual page always pointing to latest version's `introduction.html`, AND we have the `introduction.html` embedded as part of the `documentation.html` page. As a result you could have these two pages actually pointing to the same doc:

```
https://kafka.apache.org/{latest-version}/documentation.html#introduction
https://kafka.apache.org/intro
```

Ideally speaking, in the first case the ref links should be pointing to a versioned page; and in the second case the ref links should be pointing to the latest version. For now maybe we'd just follow the pattern as if both are for the latter case.